hieradata_aws/class/staging/db_admin.yaml

```diff
@@ -117,6 +117,25 @@ govuk_env_sync::tasks:
   "pull_licensify_refdata_production_daily":
     <<: *pull_licensify
     database: "licensify-refdata"
+  "push_licensify_staging_daily": &push_licensify_staging
+    ensure: "present"
+    hour: "5"
+    minute: "15"
+    action: "push"
+    dbms: "documentdb"
+    storagebackend: "s3"
+    database: "licensify"
+    temppath: "/tmp/licensify_staging"
+    url: "govuk-staging-database-backups"
+    path: "mongo-licensing"
+  "push_licensify_audit_staging_daily":
```

Comment:
I'm not sure it's worth pushing the audit table, since it doesn't affect functionality as far as I'm aware. Might actually be better not to, since we don't really want all that PII in Integration.